src/style.css

```diff
@@ -0,0 +1,89 @@
+@font-face {
+  font-family: "Avenir Book";
+  src: url("./fonts/Avenir-Book.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir Heavy";
+  src: url("./fonts/Avenir-Heavy.ttf");
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url("./fonts/Avenir.ttc");
+  font-style: normal;
+  font-weight: 300;
+}
+
+body {
+  margin-top: 20px;
+}
+
+.input {
+  font-family: Avenir, sans-serif;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+  margin-bottom: 20px;
+  font-size: 16px;
+  line-height: 22px;
+  height: 66px;
+  width: calc(100% - 62px);
+  color: #3d4e61;
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: left 26px bottom 25px;
+  padding-left: 62px;
+}
+
+.input:hover {
+  font-family: Avenir, sans-serif;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: left 26px bottom 25px;
+}
+
+.input:focus {
+  font-family: Avenir, sans-serif;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+  background-image: url("./images/Search.svg");
+  background-repeat: no-repeat;
+  background-position: left 26px bottom 25px;
+}
```

Comment:
There are some repeated attributes, try to avoid code duplication by combining classes -> input, input:hover, input:focus {...}